pulley/src/lib.rs

```diff
@@ -477,6 +477,11 @@ macro_rules! for_each_op {
             f32_from_f64 = F32FromF64 { dst: FReg, src: FReg };
             /// `(st) = promote(low32(src))`
             f64_from_f32 = F64FromF32 { dst: FReg, src: FReg };
+
+            /// `low32(dst) = abs(low32(src1)) * sign(low32(src2))`
+            fcopysign32 = FCopySign32 { dst: FReg, src1: FReg, src2: FReg };
+            /// `dst = abs(src1) * sign(src2)`
+            fcopysign64 = FCopySign64 { dst: FReg, src1: FReg, src2: FReg };
```

Comment:
Ah as another comment, mind using `operands: BinaryOperands<FReg>` here? That'll match how other binops work and make this a bit more compact. Additionally mind updating the docs here? (looks copy/pasted from elsewhere)